paasta_tools/contrib/ide_helper.py

```diff
@@ -18,7 +18,19 @@ def merge_vscode_settings(file: str, setting_dict: Dict[str, Any]) -> None:
         settings = {}
 
     # update settings with the default configurations we want each file to have
-    settings.update(setting_dict)
+    for key, value in setting_dict.items():
+        if key in settings:
+            if value not in settings[key]:
+                for i, dict_entry in enumerate(setting_dict[key]):
+                    if dict_entry not in settings[key]:
+                        if isinstance(settings[key], list):
+                            settings[key].append(dict_entry)
+                        else:
+                            temp_list = [settings[key]]
+                            temp_list.append(dict_entry)
+                            settings[key] = temp_list
+        else:
+            settings[key] = value
```

Comment:
not a blocker, but it'd be nice to make this a `merge_settings` or `merge_dicts` function or something :p